fdbserver/workloads/MetaclusterManagementWorkload.actor.cpp

```diff
@@ -1024,7 +1024,12 @@ struct MetaclusterManagementWorkload : TestWorkload {
 
 			ASSERT(self->metaclusterCreated);
 			ASSERT(!exists);
-			ASSERT(hasCapacity);
+			ASSERT(ignoreCapacityLimit || hasCapacity);
+			if (!hasCapacity) {
+				ASSERT(!tenantGroupExists);
+				ASSERT(ignoreCapacityLimit);
+				ASSERT(!assignClusterAutomatically);
```

Comment:
I think this last one could be extracted from the if block as a more general assertion:

```
ASSERT(!ignoreCapacityLimit || !assignClusterAutomatically);
```

Reply:
Done